src/actions/auth.js

```diff
@@ -0,0 +1,51 @@
+import axios from "axios";
+
+export const USER_CREATED = "USER_CREATED";
+
+export const LOGIN_SUCCESS = "LOGIN_SUCCESS";
+
+const baseUrl = "http://localhost:4000";
```

Comment:
If you are using axios a nice thing to do is create a file in your src where you configure axios and then export it. After that, from your action files you import axios from this file you created and not from the library, the file looks like:

```
// src/axios.js
import axios from "axios";

const instance = axios.create({
  baseURL: "http://localhost:4000"
});
export default instance;
```